en/08_Changelogs/6.0.0.md

```diff
@@ -70,6 +71,110 @@ All other models used `SearchableDropdownField` for `has_one` relations and `Gri
+
+#### What if I don't have time to upgrade all of my page types?
+
+If you have a lot of complex page types and extensions, upgrading all of them to account for the new scaffolding might be a large task. If you want to avoid upgrading your `getCMSFields()` and `updateCMSFields()` implementations initially, you can use the `restrictRelations` and `restrictFields` scaffolding options in the `scaffold_cmsfields_settings` configuration property for your pages. You can then declare that only the fields introduced in parent classes should be scaffolded.
+
+The following configuration can be used as a base for this workaround. It will work for all page types available in commercially supported modules. If you use page types provided in third-party modules, you may need to add configuration for those as well.
+
+> [!WARNING]
+> Note that this is explicitly intended as a temporary workaround, so that you can focus on other areas of the upgrade first, and come back to your page form fields later.
+>
+> As more community modules are upgraded to account for form field scaffolding in their page types and extension classes, you may
+> need to add more fields to this list. To avoid having to continuously update these lists it's recommended that you take the time
+> to update your `getCMSFields()` and `updateCMSFields()` implementations as soon as you have time to do so.
+
+```yml
+SilverStripe\CMS\Model\SiteTree:
+  scaffold_cmsfields_settings:
+    restrictRelations:
+      # This will stop all has_many and many_many relations from being
+      # scaffolded except for new relations which are added to this list
+      - 'ThisRelationDoesntExist'
+    restrictFields:
+      # These fields are scaffolded from SiteTree, and are the bare minimum
+      # fields that we need to be scaffolded for all page types
+      - 'Title'
+      - 'MenuTitle'
+      - 'URLSegment'
+      - 'Content'
+
+SilverStripe\CMS\Model\VirtualPage:
+  scaffold_cmsfields_settings:
+    restrictFields:
+      - 'CopyContentFrom'
+
+SilverStripe\CMS\Model\RedirectorPage:
+  scaffold_cmsfields_settings:
+    restrictFields:
+      - 'ExternalURL'
+      - 'LinkTo'
+      - 'LinkToFile'
+
+SilverStripe\Blog\Model\BlogPost:
+  scaffold_cmsfields_settings:
+    restrictRelations:
+      - 'Categories'
+      - 'Tags'
+    restrictFields:
+      - 'Summary'
+      - 'FeaturedImage'
+      - 'PublishDate'
+
+SilverStripe\IFrame\IFramePage:
+  scaffold_cmsfields_settings:
+    restrictFields:
+      - 'ForceProtocol'
+      - 'IFrameURL'
+      - 'IFrameTitle'
+      - 'AutoHeight'
+      - 'AutoWidth'
+      - 'FixedHeight'
+      - 'FixedWidth'
+      - 'BottomContent'
+      - 'AlternateContent'
+
+SilverStripe\UserForms\Model\UserDefinedForm:
+  scaffold_cmsfields_settings:
+    restrictRelations:
+      - 'EmailRecipients'
+```
```

Comment:
This yaml snippet is pretty long - if you want I can put it inside a collapsed `<details>` block.

Reply:
Yeah collapsible would be good.

Reply:
Done